src/devices/video/hp1ll3.cpp

```diff
@@ -569,17 +569,19 @@ WRITE8_MEMBER( hp1ll3_device::write )
 		switch (m_command)
 		{
 		case CONF:
-			assert((m_conf_ptr >> 1) < ARRAY_LENGTH(m_conf));
-			if (m_conf_ptr & 1) {
-				m_conf[m_conf_ptr >> 1] |= data;
-			} else {
-				m_conf[m_conf_ptr >> 1] = data << 8;
+			if (m_conf_ptr < 2 * ARRAY_LENGTH(m_conf)) {
+				if (m_conf_ptr & 1) {
+					m_conf[m_conf_ptr >> 1] |= data;
+				} else {
+					m_conf[m_conf_ptr >> 1] = data << 8;
+				}
+				m_conf_ptr++;
 			}
-			if (m_conf_ptr++ == 22) {
-				DBG_LOG(2,"HPGPU",("CONF data received: %04X %04X %04X %04X %04X %04X %04X %04X %04X %04X %04X\n",
+			if (m_conf_ptr == 2 * ARRAY_LENGTH(m_conf)) {
+				DBG_LOG(1,"HPGPU",("CONF data received: %04X %04X %04X %04X %04X %04X %04X %04X %04X %04X %04X %04X\n",
```

Comment:
Oh I see, it's supposed to print the message when it gets additional `CONF` data it can't deal with.  To make things clearer, can you make the log message explicit that it's more data than it knows what to do with, and also keep incrementing `m_conf_ptr` and add it to the message so it's clear how much extra `CONF` it's received?

Reply:
exactly the opposite in fact, it prints valid data only.   good idea, though, will do that